packages/client/wallets/smart-wallet/src/api/CrossmintWalletService.ts

```diff
@@ -1,19 +1,20 @@
-import { SmartWalletChain } from "@/blockchain/chains";
-import { SignerData, StoreSmartWalletParams } from "@/types/API";
-import type { UserParams } from "@/types/Config";
-import { API_VERSION } from "@/utils/constants";
-import { UserOperation } from "permissionless";
-import { GetEntryPointVersion } from "permissionless/_types/types";
-import { EntryPoint } from "permissionless/types/entrypoint";
+import type { UserOperation } from "permissionless";
+import type { EntryPoint, GetEntryPointVersion } from "permissionless/types/entrypoint";
 
 import { blockchainToChainId } from "@crossmint/common-sdk-base";
 
+import type { SmartWalletChain } from "../blockchain/chains";
+import { CrossmintServiceError } from "../error";
+import type { UserParams } from "../types/params";
+import { SmartWalletConfigSchema } from "../types/schema";
+import type { SmartWalletConfig, StoreSmartWalletParams } from "../types/service";
 import { bigintsToHex, parseBigintAPIResponse } from "../utils/api";
+import { API_VERSION } from "../utils/constants";
 import { BaseCrossmintService } from "./BaseCrossmintService";
 
 export class CrossmintWalletService extends BaseCrossmintService {
-    async idempotentCreateSmartWallet(user: UserParams, input: StoreSmartWalletParams) {
-        return this.fetchCrossmintAPI(
+    async idempotentCreateSmartWallet(user: UserParams, input: StoreSmartWalletParams): Promise<void> {
```

Comment:
tiny thing to make sure we're not relying on the data from here until there's validation